packages/server/tests/utils/TestUtils.ts

```diff
@@ -443,6 +446,7 @@ export function initTestUtils(): TestUtilsReturnedValue {
             nickname: userNickname ?? unique(() => random.word()),
             email,
             password,
+            confirmedEmailAt: emailIsNotConfirmed ? undefined : DateTime.now(),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏We should discuss which behaviour we want to keep as I made a change to this function in one of my PRs.